cypress/integration/user_can_sign_up_spec.js

```diff
@@ -1,4 +1,4 @@
-describe("Registration", () => {
+describe.skip("Registration", () => {
   it("A user signs up and is redirected to sign in", () => {
     cy.signUp();
     cy.url().should("include", "/sessions/new");
```

Comment:
Why are these tests being skipped? Sorry I might have missed something from our earlier conversation

Reply:
Ah posted about this in Slack, have since updated